### PR TITLE
Completed dom-querying exercise

### PR DESCRIPTION
**Pull request by kaitlin-lovrich**
image

### CODE REVIEW

File: dom-querying/dom-query-quiz-notes.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
+# dom-query-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- Why do we log things to the console?
+  to test scripts
+- What is a "model"?
+
```
Comment (shawnkost) [inline dom-querying/dom-query-quiz-notes.md]: Missing a quiz question

**Review by shawnkost — CHANGES_REQUESTED** [review]
Missing 1 quiz question

File: dom-querying/dom-query-quiz-notes.md (hunk 1 above)
Comment (kaitlin-lovrich) [reply dom-querying/dom-query-quiz-notes.md]: Right you are

Comment (kaitlin-lovrich) [reply dom-querying/dom-query-quiz-notes.md]: image

File: dom-querying/dom-query-quiz-notes.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ After completing this exercise, you should be able to discuss or answer the foll
 - Why do we log things to the console?
   to test scripts
 - What is a "model"?
-
+  a model is a replica of the html
```
Comment (shawnkost) [inline dom-querying/dom-query-quiz-notes.md]: It's moreso asking in general, so more like just a "recreation of something". In the case of the DOM, it's a recreation of our HTML page, but in general, it just refers to a recreation of something

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**